api/src/environment.js

```diff
@@ -1,6 +1,20 @@
-const url = require('url'),
-      logger = require('./logger'),
-      { COUCH_URL, UNIT_TEST_ENV } = process.env;
+const path = require('path');
+const url = require('url');
+const logger = require('./logger');
+
+const { env } = process;
+const { UNIT_TEST_ENV, COUCH_URL } = env;
```

Comment:
Add `MEDIC_API_RESOURCE_PATH` and `NODE_ENV` to this Object to keep access consistent.

Reply:
Done